swift_browser_ui_frontend/src/components/CFooter.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div id="c-footer">
+    <nav class="navbar">
+    <div class="navbar-menu is-active">
+        <div class="navbar-start" >
+          <div class="col-3">
+            <div class="row navbar-item " >
+              <span class="largetext">
+              {{ $t("message.footerMenu.groupandprogram") }}
+            </span>
+            </div>
+            <div class="row navbar-item smalltext">
+              <a class="linktext"
+              href="https://csc.fi"
+              :alt="$t('message.footerMenu.csc')"
+            >{{
+              $t("message.footerMenu.csc")
+            }}</a>
+            </div>
+          </div>        
+        </div>
+    
+        <div class="navbar-end" id="rightmenu"> 
+            <div class="navbar-item smalltext">
+                <a class="linktext"
+              href="https://research.csc.fi/-/sd-connect"
+              :alt="$t('message.footerMenu.servicedescription')"
+            >{{
+              $t("message.footerMenu.servicedescription")
+            }}</a>
+          </div> 
```

Comment:
Also as an example, this part should have correct indentation, space and lines as well :)